src/index.js

```diff
@@ -3,11 +3,13 @@ import express from 'express';
 
 // use "require" to import JSON files
 const admins = require('./data/admins.json');
+const employeeRouter = require('./resources/employees');
 
 const app = express();
 const port = process.env.PORT || 3000;
 
 app.use(express.json());
+app.use('/employee', employeeRouter);
```

Comment:
let's put this prefix in plural
```suggestion
app.use('/employees', employeeRouter);
```